tests/general.test.ts

```diff
@@ -336,31 +276,6 @@ test.skip(`extra fetch options`, async () => {
   `)
 })
 
-test(`case-insensitive content-type header for custom fetch`, async () => {
```

Comment:
This was testing internal behavior of `Headers`, so this should no longer be needed.